Apps/Playground/CMakeLists.txt

```diff
@@ -74,6 +74,10 @@ elseif(UNIX)
     set(SOURCES
         ${SOURCES}
         "X11/App.cpp")
+elseif(EMSCRIPTEN)
+    set(SOURCES
+        ${SOURCES}
+        "Emscripten/App.cpp")
```

Comment:
It's a bit strange to call this Emscripten when it targets specifically GLFW. I only see `emscripten_set_main_loop_arg` that is Emscripten specific in App.cpp.